### ISSUE TITLE
Uncaught SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by neontapir** [TimKam/atomic-vale#11]
[Enter steps to reproduce:]

1. Configured Vale with a .vale file

StylesPath=/Users/ME/Documents/vale-styles
MinAlertLevel = warning

[*]
BasedOnStyles = 18F
18F.UnexpandedAcronyms = NO

2. Make any change to a particular Asciidoc file. Others work fine.

**Atom**: 1.21.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [atomic-vale](https://github.com/TimKam/atomic-vale) package 1.6.0

### Stack Trace

Uncaught SyntaxError: Unexpected end of JSON input

```
At file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.html:395

SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /packages/atomic-vale/lib/init.coffee:91:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:551:20)
```

### Commands

```
     -7:11.3.0 core:delete (input.hidden-input)
     -7:05.0 core:backspace (input.hidden-input)
  2x -7:04.4.0 core:move-right (input.hidden-input)
  2x -7:03.8.0 core:backspace (input.hidden-input)
     -5:50.7.0 core:move-up (input.hidden-input)
  2x -5:40.1.0 core:save (input.hidden-input)
  4x -4:29.6.0 core:backspace (input.hidden-input)
     -2:41.2.0 core:save (input.hidden-input)
     -2:12.3.0 linter-ui-default:toggle-panel (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
  2x -2:07.3.0 core:save (input.hidden-input)
     -1:28.3.0 linter-ui-default:toggle-panel (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
  3x -1:07.8.0 core:save (input.hidden-input)
     -0:11.1.0 markdown-writer:insert-new-line (input.hidden-input)
     -0:11.1.0 editor:newline (input.hidden-input)
     -0:10.5.0 editor:move-to-end-of-screen-line (input.hidden-input)
  2x -0:10.4.0 core:delete (input.hidden-input)
```

### Non-Core Packages

```
asciidoc-assistant 0.2.2 
asciidoc-image-helper 1.0.0 
asciidoc-preview 2.10.2 
atom-beautify 0.30.5 
atomic-vale 1.6.0 
autocomplete-asciidoc 0.1.2 
busy-signal 1.4.3 
change-case 0.6.5 
doc-green-syntax 1.1.0 
doc-green-ui 1.1.0 
document-outline 1.6.13 
fascination-syntax 15.5.34 
intentions 1.1.5 
language-asciidoc 1.10.0 
language-markdown 0.25.1 
lines 0.13.1 
linter 2.2.0 
linter-spell 0.15.0 
linter-ui-default 1.6.10 
markdown-format 2.5.1 
markdown-preview-plus 2.4.10 
markdown-themeable-pdf 1.2.0 
markdown-writer 2.7.2 
minimap 4.29.7 
pdf-view 0.59.0 
pigments 0.40.2 
pretty-json 1.6.4 
print-atom 0.1.1 
set-syntax 0.4.0 
smart-quotes-plus 2.1.0 
spell-check-test 0.77.5 
status-stats-jbrains 0.6.0 
tidy-markdown 3.0.1 
yard 0.3.1 
```

**Comment by TimKam** [comment]
Thanks for reporting this. There seems to be an issue with files that exceed a specific lengths (somewhere there's an input length limit that's then exceeded). I'll take a look over the weekend.

**Comment by TimKam** [comment]
Fixed with [v1.7.0](https://github.com/TimKam/atomic-vale/releases/tag/v1.7.0).
Sorry that it took me longer than expected to find the time.
